lib/assert.js

```diff
@@ -150,6 +152,14 @@ x.ifError = (err, msg) => {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.jsxEqual = (val, expected, msg) => {
+	test(deepEqual(val, expected), create(val, expected, '===', msg, x.jsxEqual));
+};
+
+x.notJsxEqual = (val, expected, msg) => {
+	test(!deepEqual(val, expected), create(val, expected, '!==', msg, x.notJsxEqual));
+};
```

Comment:
Ok. I like that idea. Can you remove the `jsxEqual` stuff for now and open a new issue about `jsxEqual`? We can add it back when we have a better story for it. It was pretty unrelated to this pull request anyways.